dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'greenhouse_source'
-version: '0.1.0'
+version: '0.1.1'
```

Comment:
If we update the package to include out custom schema logic we should update this to v0.2.0 as it would be a breaking change

Reply:
good call -- switched to 0.2.0